cliqz_env.sh

```diff
@@ -49,9 +49,10 @@ else
   MAKE=make
 fi
 
-# TODO: Use MOZ_UPDATE_CHANNEL.
 if [ -z $CQZ_RELEASE_CHANNEL ]; then
-  export CQZ_RELEASE_CHANNEL=release
+  export MOZ_UPDATE_CHANNEL=beta
+else
+  export MOZ_UPDATE_CHANNEL=$CQZ_RELEASE_CHANNEL
```

Comment:
This inverses the logic of `CQZ_RELEASE_CHANNEL`, meaning that current Jenkins configurations will have to be updated.